.github/workflows/update-trivy-cache.yml

```diff
@@ -0,0 +1,41 @@
+name: Update Trivy Cache
+
+on:
+  schedule:
+    - cron: '0 0 * * *'  # Run daily at midnight UTC
+  workflow_dispatch:
+
+permissions: read-all
+
+jobs:
+  update-trivy-db:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup oras
+        uses: oras-project/setup-oras@9c92598691bfef1424de2f8fae81941568f5889c # v1.2.1
+
+      - name: Get current date
```

Comment:
what happens to workflows that rely upon cache of that day when this one fails? Do they fail back to downloading DB?

Reply:
Nope, it will fail the scan since I add `skipDBUpdate` flag. That's why I retry 10 times to ensure it does download the DB. One optimization could be retrying every minute (or 5 min) until the cache is downloaded. I just want to completely remove the download process from scan step.

And in the worst case, we can manually trigger the job to update the cache.

Reply:
sounds good